packages/web/src/components/Grid/_tools.scss

```diff
@@ -0,0 +1,34 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+@use '../../tools/breakpoint';
+
+@function breakpointClass($name, $value) {
+    @if ($value > 0) {
+        @return unquote($name + '--');
+    }
+
+    @return '';
+}
+
+@mixin gutters($gutters, $breakpoints) {
+    @each $breakpointName, $breakpointValue in $breakpoints {
```

Comment:
We should unify the syntax of variable, mixin and function names. I'd stick with kebab-case since it's already used in our design tokens.